app/eq.py

```diff
@@ -128,6 +130,29 @@ async def build(self):
 
         return self._payload
 
+    @staticmethod
+    def encrypt_response_id(questionnaire_id: str):
+        """
+        Encrypts a response_id using the following format: <response_id><first 16 characters of sha256 signature>.
+        Utilises EQ_SALT environment property as salt.
+        Implements the HMAC algorithm as described by RFC 2104.
+
+        :param questionnaire_id: the questionnaire_id utilised to encrypt the response_id
+        :return: the encrypted response_id
+        """
+        try:
+            eq_salt = app.config.BaseConfig.EQ_SALT
+            qid = questionnaire_id.encode()
+            eqs = eq_salt.encode()
+
+            hashed_password = hashlib.sha256(eqs + qid).hexdigest()
+            hp_16_chars = hashed_password[0:16]
+            response_id = questionnaire_id + hp_16_chars
+
+            return response_id
+        except ValueError:
```

Comment:
I don't think this exception will ever trigger. I don't think it is in the code-coverage.  Remove the try-catch.